end_to_end/tpu/mixtral/8x7b/2_test_mixtral.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# This file, combined with step 1 in the same directory, demonstrates:
```

Comment:
Perhaps mention this test is run at a nightly cadence

https://github.com/google/maxtext/blob/db31dd4b0b686bca4cd7cf940917ec372faa183a/end_to_end/tpu/gemma/7b/2_test_gemma.sh#L3

Reply:
Sure, I will emphasize it's a nightly run.

Reply:
Added `runs on daily basis`